Library/Formula/vile.rb

```diff
@@ -0,0 +1,23 @@
+class Vile < Formula
+  desc "Vi Like Emacs"
+  homepage "http://invisible-island.net/vile/vile.html"
+  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
+  version "9.8"
+  sha256 "dea31c023aa7a90271056c712c826a5d16d19ad61df5cf5f299aa1025af280d6"
+
+  depends_on "reflex"
```

Comment:
Hmm. When I attempt to do this, I get a 403 'Permission denied' error:

/usr/local$ git push --set-upstream origin adding_vile
remote: Permission to Homebrew/homebrew.git denied to timbck2.
fatal: unable to access 'https://github.com/Homebrew/homebrew/': The
requested URL returned error: 403

Tim Buck
Shared joy is a double joy; shared sorrow is half a sorrow. - Traditional
Swedish proverb

On Thu, Jan 7, 2016 at 3:21 AM, Baptiste Fontaine <notifications@github.com>
wrote:

> In Library/Formula/vile.rb
> <https://github.com/Homebrew/homebrew/pull/47698#discussion_r49059254>:
>
> > @@ -0,0 +1,23 @@
> > +class Vile < Formula
> > +  desc "Vi Like Emacs"
> > +  homepage "http://invisible-island.net/vile/vile.html"
> > +  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
> > +  version "9.8"
> > +  sha256 "dea31c023aa7a90271056c712c826a5d16d19ad61df5cf5f299aa1025af280d6"
> > +
> > +  depends_on "reflex"
>
> Now make sure you’re on your branch (git checkout adding_vile) then do git
> push to push these changes on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/47698/files#r49059254>.
>